docs/ecosystem-vue.md

```diff
@@ -7,45 +7,51 @@ sidebar_label: Vue
 single-spa-vue is a helper library that helps implement [single-spa registered application](configuration#registering-applications) [lifecycle functions](building-applications.md#registered-application-lifecycle) (bootstrap, mount and unmount) for for use with [Vue.js](https://vuejs.org/). Check out the [single-spa-vue github](https://github.com/CanopyTax/single-spa-vue).
 
 ## Starter repo
+
 For a full example, see [coexisting-vue-microfrontends](https://github.com/joeldenning/coexisting-vue-microfrontends).
 
 ## Live demo
+
 https://coexisting-vue-microfrontends.surge.sh
 
 ## Installation
+
 ### Vue CLI
+
 The [vue-cli-plugin-single-spa](https://github.com/CanopyTax/vue-cli-plugin-single-spa) will get everything set up.
 
 ```sh
 vue add single-spa
 ```
 
 The CLI Plugin does the following for you:
-1) Modify your webpack config so that your project works as a single-spa application or parcel.
-2) Install [single-spa-vue](https://github.com/CanopyTax/single-spa-vue).
-3) Modify your `main.js` or `main.ts` file so that your project works as a single-spa application or parcel.
-4) Add a `set-public-path.js` that will use `systemjs-webpack-interop` in order to set the public path of your application.
+
+1. Modify your webpack config so that your project works as a single-spa application or parcel.
+2. Install [single-spa-vue](https://github.com/CanopyTax/single-spa-vue).
+3. Modify your `main.js` or `main.ts` file so that your project works as a single-spa application or parcel.
+4. Add a `set-public-path.js` that will use `systemjs-webpack-interop` in order to set the public path of your application.
```

Comment:
can we change these to just `1.` for all of them? that way we don't need to keep the list ordered in future revisions. :)

Reply:
Sure, I think it's ok